drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/wizard/column/NewGuidedDecisionTableColumnWizard.java

```diff
@@ -0,0 +1,195 @@
+        return this.pages;
+    }
+
+    public void setPages(final List<WizardPage> pages) {
+        this.pages = pages;
+    }
+
+    @Override
+    public Widget getPageWidget(final int pageNumber) {
+        final WizardPage wizardPage = this.pages.get(pageNumber);
+        final Widget widget = wizardPage.asWidget();
+
+        wizardPage.prepareView();
+
+        return widget;
+    }
+
+    @Override
+    public int getPreferredHeight() {
+        return 600;
+    }
+
+    @Override
+    public int getPreferredWidth() {
+        return 900;
+    }
+
+    @Override
+    public void isComplete(final Callback<Boolean> callback) {
+        //Assume complete
+        callback.callback(true);
+
+        for (WizardPage page : this.pages) {
+            page.isComplete(result -> {
+                if (Boolean.FALSE.equals(result)) {
+                    callback.callback(false);
+                }
+            });
+        }
+    }
+
+    @Override
+    public void start() {
+        start(new ArrayList<>());
+    }
+
+    public void start(List<WizardPage> pages) {
+        this.pages = new ArrayList<WizardPage>() {{
+            add(summaryPage);
+            addAll(pages);
+        }};
+
+        for (WizardPage page : this.pages) {
+            ((BaseDecisionTableColumnPage) page).init(this);
+        }
+
+        super.start();
+    }
+
+    public void start(final BaseDecisionTableColumnPlugin plugin) {
+        plugin.init(this);
+
+        loadPages(plugin);
+        initPages(plugin);
+
+        super.start();
+    }
+
+    private void initPages(final BaseDecisionTableColumnPlugin plugin) {
+        for (final WizardPage page : pages) {
+            final BaseDecisionTableColumnPage tableColumnPage = (BaseDecisionTableColumnPage) page;
+
+            tableColumnPage.init(this);
+            tableColumnPage.setPlugin(plugin);
+        }
+    }
+
+    private void loadPages(final BaseDecisionTableColumnPlugin plugin) {
+        pages.clear();
+        pages.add(summaryPage);
+        pages.addAll(plugin.getPages());
+    }
+
+    @Override
+    public void complete() {
+        if (finishCommand.get()) {
+            super.complete();
+        }
+    }
+
+    @Override
+    public void close() {
+        onCloseCallback.execute();
+
+        super.close();
+    }
+
+    private WizardView getView() {
+        return view;
+    }
+
+    public void goTo(final int index) {
+        getView().selectPage(index);
+    }
+
+    public void init(final GuidedDecisionTableView.Presenter presenter) {
+        this.presenter = presenter;
+    }
+
+    public void setFinishCommand(final Supplier<Boolean> finishCommand) {
+        this.finishCommand = finishCommand;
+    }
+
+    public void setOnCloseCallback(final Command onCloseCallback) {
+        this.onCloseCallback = onCloseCallback;
+    }
+
+    public GuidedDecisionTableView.Presenter getPresenter() {
+        return presenter;
+    }
+
+    private String translate(final String key,
```

Comment:
I've noticed this too. But all alternatives that I see make the code prolix. Since the translate does not do anything, just keeps the code easier to read, I think that it's ok to keep that as it is. But thanks :-)